inductiva/simulation/simulator.py

```diff
@@ -81,5 +87,28 @@ def run_async(
         return api.run_async_simulation(
             self.api_method_name,
             input_dir,
-            params=kwargs,
+            params=[kwargs],
+        )
+
+    def run_pipeline(self,
+                     input_dir: types.Path,
+                     *_args,
+                     output_dir: Optional[types.Path] = None,
+                     pipeline: List[Command],
+                     track_logs: bool = False) -> pathlib.Path:
+        """Run a pipeline of commands.
+        Args:
+            working_dir: Path to the directory where the pipeline will be 
+            executed.
+            pipeline: List of commands to be executed in the simulation API.
+        """
+        input_dir, output_dir = self._setup_dirs(input_dir, output_dir)
+
+        params = [command.get_args() for command in pipeline]
+        return api.run_simulation(
+            self.api_method_name,
```

Comment:
I didn't think about this parameter. But I agree that I need to test this to see where I will get into problems. Because I would assume it is that for gromacs but I need to check.

Reply:
I'm sorry, my mistake, this parameter is not referring to the simulation at all, just with the simulator we are using. So in principle, you are assuming command contains everything.